discovery-streams/values.yaml

```diff
@@ -1,26 +1,35 @@
+global:
+  kafka:
+    brokers: pipeline-kafka-bootstrap:9092
+  redis:
+    host: redis.external-services
+    password: ""
+  ingress:
+    dnsZone: ""
+    rootDnsZone: ""
+    port: 80
+    scheme: ""
+    subnets: ""
+    security_groups: ""
```

Comment:
Are all these ingress values still used with the annotation changes?

Reply:
Same comment as above, we are deciding to remove the scheme, subnets and security groups and opting for people to put those in their specific values files as needed.